File: john-otienoh/power_of_two.c
#include <stdio.h>
char *power_of_two(int n);
/**
 * main - Entry point of the function
 * Return: Always 0(success)
 */
int main(void)
{
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);

    (power_of_two(n) == "True") ? printf("True\n") : printf("False\n");
    return (0);
}

/**
 * power_of_two - Program that takes an integer as input
 * @n: Integer
 * Return: true if the input is a power of two else false. 
 */

char *power_of_two(int n)
{
    while (n % 2 == 0)
    {
        n = n / 2;
        if (n == 1)
            return ("True");
    }
}

File: john-otienoh/power_of_two.py
#!/usr/bin/python3

def power_of_two(n):
    """
        program that takes an integer as input
        returns true if the input is a power of two else false.
    """
    return False if n <= 0 else (n & (n - 1) ) == 0


if __name__ == "__main__":
    print(power_of_two(int(input("Enter Number: "))))
    